blurb

```diff
@@ -650,24 +650,36 @@ Run unit tests.  Only works inside source repo, not when installed.
     print(tests_run, "tests passed.")
 
 
+def find_editor():
+    for var in 'GIT_EDITOR', 'EDITOR':
+        editor = os.environ.get(var)
+        if editor is not None:
+            return editor
+    if sys.platform == 'win32':
+        fallbacks = ['notepad.exe']
+    else:
+        fallbacks = [
+            '/etc/alternatives/editor',
+            '/usr/bin/editor',
```

Comment:
I don't want "/usr/bin/editor" or "editor" either.  If they have this Debian "alternatives" thing, then /etc/alternatives/editor will be valid, and if not I don't want to pick up random junk.  The only fallbacks I want for POSIX are "/etc/alternatives/editor" and "nano".